examples/xnnpack/aot_compiler.py

```diff
@@ -72,6 +73,8 @@
         *MODEL_NAME_TO_MODEL[args.model_name]
     )
 
+    sys.setrecursionlimit(30000)
```

Comment:
alternatively you can do
```
with validation_disabled():
    to_backend(model, XnnpackPartitioner)
```

To turn off the check which is hitting recursion limit